Change the default to 0, which is a no-op for data_bitrate.

dcasnowdon opened the pull request:
<!--
Thank you for contributing to python-can!
Please fill out the following template to help us review your pull request.
-->

## Summary of Changes

<!-- Briefly describe what your pull request does. -->

- 

## Related Issues / Pull Requests

<!-- List any related issues, pull requests, or discussions. -->

- Closes #
- Related to #

## Type of Change

- [ ] Bug fix
- [ ] New feature
- [ ] Documentation update
- [ ] Refactoring
- [ ] Other (please describe):

## Checklist

- [ ] I have followed the [contribution guide](https://python-can.readthedocs.io/en/main/development.html).
- [ ] I have added or updated tests as appropriate.
- [ ] I have added or updated documentation as appropriate.
- [ ] I have added a [news fragment](doc/changelog.d/) for towncrier.
- [ ] All checks and tests pass (`tox`).

## Additional Notes

<!-- Add any other information or context you want to share. -->

zariiii9003 commented:
Hi, thank you for the fix. But I'd prefer the function signature as it is, so instead you could change the `_DATA_BITRATES` value to `None: ""`

dcasnowdon commented:
> Hi, thank you for the fix. But I'd prefer the function signature as it is, so instead you could change the `_DATA_BITRATES` value to `None: ""`

I wasn't sure whether 0 might be passed in as a no-op. Is it OK to change the interface in this way? Instead, I've changed it to add an explicit check for `None` in the body of the method. I can change the value in the `_DATA_BITRATES` table if you'd prefer.

dcasnowdon commented:
> Yes, looks good now. Could you add a little bit more information to the news fragment? I'll create a patch release afterwards

I have added a little more detail (is this what you'd like?), and I've changed it to be a `fix`, rather than `added`.
